app/src/main/java/com/example/androidacademyhomework/UI/movielist/MoviesListFragmentViewModel.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.androidacademyhomework.UI.movielist
+
+import androidx.lifecycle.*
+import androidx.paging.*
+import com.example.androidacademyhomework.createMovie
+import com.example.androidacademyhomework.data.model.Movie
+import com.example.androidacademyhomework.repository.MovieRepository
+import kotlinx.coroutines.flow.*
+
+class MoviesListFragmentViewModel(private val repository: MovieRepository) : ViewModel() {
+
+    private var currentQueryValue: String? = null
+    private var currentSearchResult: Flow<PagingData<Movie>>? = null
+    private var currentResult: Flow<PagingData<Movie>>? = null
+
+   private val _isSearch = MutableStateFlow<Boolean>(false)
+     val isSearch: StateFlow<Boolean>
+     get() = _isSearch
+
+    fun popularMovies(): Flow<PagingData<Movie>> {
+        _isSearch.value = true
```

Comment:
false?

Reply:
Not the correct name, in this way I checked whether the first launch of the application. Fixed